packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx

```diff
@@ -230,26 +230,31 @@ export const menuTabs: Record<ForumTypeString,Array<MenuTab>> = {
       showOnMobileStandalone: true,
       showOnCompressed: true,
     }, {
-      id: 'sequences',
-      title: 'Sequences',
-      link: '/sequences',
+      id: 'library',
+      title: 'Library',
+      link: '/library',
       iconComponent: LocalLibrary,
-      tooltip: "Collections of posts building on a common theme",
+      tooltip: "Core reading, and sequences of posts building on a common theme",
       showOnMobileStandalone: true,
       showOnCompressed: true,
-    // TODO: Once we get two sequences we want to share, add subitems
-    // }, {
-    //   id: 'fellowship-reading',
-    //   title: 'Fellowship Reading',
-    //   link: '/TODO:',
-    //   tooltip: "TODO: James is cool",
-    //   subItem: true,
-    // }, {
-    //   id: 'motivation',
-    //   title: 'Motivation',
-    //   link: '/TODO:',
-    //   tooltip: "TODO: Aaron's cool",
-    //   subItem: true,
+    }, {
+      id: 'handbook',
+      title: 'The EA Handbook',
+      link: '/handbook',
```

Comment:
Is this the right link? Aaron has a comment in this post saying it's an older version: https://forum.effectivealtruism.org/handbook?commentId=MRRQYrhFemqzTSoa8

Reply:
I think Aaron's language there is legit confusing. FWIW.